frame/fee-market/src/lib.rs

```diff
@@ -68,14 +66,16 @@ pub mod pallet {
 		type TreasuryPalletId: Get<PalletId>;
 		#[pallet::constant]
 		type LockId: Get<LockIdentifier>;
-		/// The minimum locked collateral for a fee market relayer, also represented as the maximum value for slash.
-		#[pallet::constant]
-		type MiniumLockCollateral: Get<RingBalance<Self>>;
+
 		/// The minimum fee for relaying.
 		#[pallet::constant]
 		type MinimumRelayFee: Get<Fee<Self>>;
+		/// The assigned relayers number for each order.
 		#[pallet::constant]
 		type AssignedRelayersNumber: Get<u64>;
+		/// The collateral relayer need to lock for each order.
+		#[pallet::constant]
+		type CollateralPerOrder: Get<RingBalance<Self>>;
```

Comment:
When we update this value in runtime upgrade, we might cause some invalidation in the correction of message capacity and collateral insufficiency. We might need to update the capacity in all relayers and invalidate some relayers(but some relayers might not be able to being removed due to working order)

One proposal will be just add the locked collateral value for each order in order struct, then there will not be issues for slashing when change collateralPerOrder. And capacity is calculated from collateral - all_locked_collateral_in_orders. (Whether or how to use cache is another question)